iree/compiler/Dialect/VMLA/Transforms/CMakeLists.txt

```diff
@@ -27,13 +27,10 @@ iree_cc_library(
     iree::compiler::Dialect::VMLA::Conversion::HLOToVMLA
     iree::compiler::Dialect::VMLA::Conversion::StandardToVMLA
     iree::compiler::Dialect::VMLA::IR
-    iree::compiler::Utils
     LLVMSupport
     MLIRIR
     MLIRPass
-    MLIRStandardOps
```

Comment:
Sure this isn't needed or mistake in bazel_to_cmake?

Reply:
Pretty sure it's not needed. If there's a mistake it's in the bazel build file itself because it doesn't have that dep either

Reply:
:+1: Didn't had time to check the Bazel file.